### PR TITLE
No materialize

### PR DESCRIPTION
**Pull request by caal-15**
Here's a component that creates and adds problems to the contest (served as the main page, that must change, but is not urgent by any means), removed all traces of materialize css as well as adding the basic grid, please review and merge at your discretion

### CODE REVIEW

File: browser/main.js [hunk 1]
```diff
@@ -1,11 +1,10 @@
 var React = require('react');
 var ReactDOM = require('react-dom');
+var ContestForm = require('./components/contest_form.js');
 
 document.addEventListener('DOMContentLoaded', function() {
   ReactDOM.render(
-    <h1>Hello, world!</h1>,
+    <ContestForm url='http://127.0.0.1:8080/' />,
```
Comment (pin3da) [inline browser/main.js]: This must be taken from a global state, for example window.config.backend_address  or something like that.

In this way you can make changes easily.

**Comment by caal-15** [comment]
If someone doesn't comment or sth I'm gonna merge this

**Comment by leiverandres** [comment]
Let me take a look

**Comment by leiverandres** [comment]
A question: do we use browser/main.js to  render the components in order to get global url?
Other things: we said that we were using ES6 react classes, like in the styleguide I added

The big deal is that the component looks pretty good for me

**Comment by pin3da** [comment]
Minor changes added, I think is ready to merge. :+1:

**Pull request merged**